packages/backend/src/managers/modelsManager.ts

```diff
@@ -225,10 +228,10 @@ export class ModelsManager implements Disposable {
     }
 
     tasks.forEach(task => {
-      if (isProgressEvent(event)) {
+      if (isProgressProgressiveEvent(event)) {
```

Comment:
Is the rename really necessary ?

Reply:
To keep in line with the naming used. It can be updated with the above if you have some suggestion

Reply:
I did in https://github.com/projectatomic/ai-studio/pull/204#discussion_r1519602557 ! Tell me what you think about it